modules/nw-infw-operator-config-object.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-node-firewall-operator.adoc
+
+:_content-type: CONCEPT
+[id="nw-infw-operator-config-object_{context}"]
+== Ingress Node Firewall Operator configuration object
+
+The fields for the Ingress Node Firewall Operator are described in the following table:
+
+.Cluster Network Operator configuration object
+[cols=".^2,.^2,.^6a",options="header"]
+|====
+|Field|Type|Description
+
+|`metadata.name`
+|`string`
+|The name of the CR object. The name of the config object must be `ingressnodefirewallconfig`.
+
+|`spec.nodeSelector`
+|`object`
+|
+A node selection constraint used to target nodes through specified node labels. For example:
+
+[source,yaml]
+----
+spec:
+  nodeSelector:
+    node-role.kubernetes.io/worker: ""
+----
+
+[NOTE]
+====
+The labels used in `nodeSelector` must match the labels on the node in order for the DaemonSet to start. For example, if node labels `node-role.kubernetes.io/worker` and `node-type.kubernetes.io/vm` are applied to a node then both labels must be set using `nodeSelector` for the DaemonSet to start.
+====
+
+|`spec.matchLabels`
+|`object`
```

Comment:
there is no matchLabels for this object 
only nodeSelector string
https://github.com/openshift/ingress-node-firewall/blob/master/api/v1alpha1/ingressnodefirewallconfig_types.go#L26

Reply:
there is no match labels for this field only string as stated earlier